site/src/reusecore/Button/btn.style.js

```diff
@@ -1,71 +1,91 @@
-import styled, { css } from "styled-components";
-const ButtonStyle = styled.button` 
-    cursor: pointer;
-    display: inline-flex;
-    align-items: center;
-    justify-content: center; 
-    font-family: inherit; 
-    font-size: 16px;
-    text-decoration: none;
-    text-transform: capitalize; 
-    border: 0; 
-    min-width: 170px;
-    padding: 14px;
-    border-radius: 5px;
-    -webkit-transition: 450ms all;
-    transition: 450ms all;
-    position: relative;
-    color: ${props => props.theme.white };
-    background-color: #00B39F;
-    z-index: 999;
-    &:hover,
-    &:focus {
-        color: white;
-        background: ${props => props.theme.activeColor}; 
-        box-shadow: 0 2px 10px ${props => props.theme.DarkTheme ? "rgb(255 255 255 / 40%)" : "rgb(0 0 0 / 40%)"};
-    }
-    &:active{
-        box-shadow: 0 2px 10px ${props => props.theme.DarkTheme ? "rgb(0 0 0 / 40%)" : "rgb(255 255 255 / 40%)"};
-        transform: scale(0.98);
-    }
-    .icon-left{
-        margin-right: 8px;
-    }
-    .icon-right{
-        margin-left: 8px;
-    }
+import styled, { css } from 'styled-components';
+const ButtonStyle = styled.button`
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-family: inherit;
+  font-size: 16px;
+  text-decoration: none;
+  text-transform: capitalize;
+  border: 0;
+  min-width: 170px;
+  padding: 14px;
+  border-radius: 5px;
+  -webkit-transition: 450ms all;
+  transition: 450ms all;
+  position: relative;
+  color: ${props => props.theme.white};
+  background-color: #00b39f;
```

Comment:
we should use theme colors where its available i guess

Reply:
Not sure I touched this file, I think it should be formatting, but yeah true theme colors should be used where necessary, for the button I think the same color was used on both light and dark modes